Automattic/wp-calypso
# Seller Experience - Add new design picker UI

#### Proposed Changes

* Changes the design picker UI 

#### Testing Instructions

* Go to `/setup/designSetup?siteSlug=[SITE-SLUG]&flags=signup/theme-preview-screen`
* On the design picker step, categories should be visible
* Clicking on a theme, you should be redirected to its preview page (the same should happen in other flows)

Closes #64715


-----

After the `signup/theme-preview-screen` feature flag is enabled everywhere, we need to remove 
```
/* Hide categories from the sell intent */
.design-picker__sell-intent {
	.design-picker-category-filter__sidebar,
	.design-picker-category-filter__dropdown,
	.design-picker__category-heading-0 {
		display: none;
	}
}
```
from `client/landing/stepper/declarative-flow/internals/steps-repository/design-setup/style.scss`
